platform/lib/build/pageTransformer.js

```diff
@@ -141,14 +141,16 @@ class PageTransformer {
    * @param  {Vinyl}
    * @return {undefined}
    */
-  _skipFilter(page, format) {
+  _isManuallyFiltered(page) {
     let path = page.path.replace('.amp.html', '.html');
 
-    // Skip pages that have been manually filtered and therfore have a path like
+    // Skip pages that have been manually filtered and therefore have a path like
     // - guides-and-tutorials/index.websites.html
     // - guides-and-tutorials/index.email.amp.html
-    if (path.match(/\.(websites|stories|ads|email)(\.amp)?\.html/)) {
-      return true;
+    // But still the matching format should be filtered, therefore pass it back
+    const format = path.match(/\.(websites|stories|ads|email)(\.amp)?\.html/);
+    if (format) {
+      return format[1];
```

Comment:
Is it intentional that only this path returns a format?

Reply:
vs true/false

Reply:
Yes and no, those should have been two separate methods as now.